src/msm.rs

```diff
@@ -297,7 +297,25 @@ pub fn multiexp_serial<C: CurveAffine>(coeffs: &[C::Scalar], bases: &[C], acc: &
         (f64::from(bases.len() as u32)).ln().ceil() as usize
     };
 
-    let number_of_windows = C::Scalar::NUM_BITS as usize / c + 1;
+    let field_byte_size = coeffs[0].as_ref().len();
```

Comment:
maybe div_ceil by 8 is better option here because we don't force consumer to send non empty vectors